src/test/daml/Daml/Finance/Settlement/Test/Batch.daml

```diff
@@ -125,12 +125,12 @@ run settleCashOnLedger = script do
     else
       do
         -- Settle by off ledger trasnfer
-        (cashInstructionCid, _) <- submit buyer do
+        (cashInstructionCid, _) <- submitMulti [buyer, bank] [] do
```

Comment:
Why is it that we need both signatures for the off-ledger mode? Was this a deliberate choice to make it so?

Reply:
I think the most conservative makes sense:

1. to `Allocate` with a `SettleOffledger`, we require both the `custodian` and `sender` to authorize it
2. to `Approve` with `SettleOffledgerAcknowledge`, we require both the `custodian` and `receiver` to authorize it

With a side contract, we can then relax it to, e.g., let the `sender` and `receiver` authorize it without the `custodian`.

Reply:
Ah right, it's because we don't have a holding or account to take the controllers from. Makes sense.